modules/33acrossBidAdapter.js

```diff
@@ -640,6 +651,31 @@ function _createSync({ siteId = 'zzz000000000003zzz', gdprConsent = {}, uspConse
   return sync;
 }
 
+function getTopMostAccessibleFrame() {
+  let mostAccessibleWindow = utils.getWindowSelf();
+
+  try {
+    while (mostAccessibleWindow.parent !== mostAccessibleWindow &&
```

Comment:
I am seeing that `refererDetection` also uses a loop (`do..while` instead of `while`), the `parent` property and a try/catch wrapping `location.href` (instead of trying to access the `document`). It's pretty similar to our logic, but unfortunately this API doesn't expose the top most accessible window :(